### PR TITLE
Removing inaccurate note about config file

### PR DESCRIPTION
**Pull request by eliskaachee**
### Background

The config file does not override command line values. So removing text that says it does.

### Changes

Removing text.

### Testing

Is text removed? Then yay!

### CODE REVIEW

File: example_panther_config.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Example configuration file for panther_analysis_tool.
 # Copy this file into the root of your test repository and name it .panther_settings.yml
-# Note that options in this file override options passed on the command line
```
Comment (darwayne) [inline example_panther_config.yml:3]: Can we just update the note to be accurate?

File: panther_analysis_tool/main.py [hunk 2]
```diff
@@ -1775,11 +1775,7 @@ def run() -> None:
         if key.startswith("PANTHER_"):
             logging.info("Found Environment Variables prefixed with 'PANTHER'.")
             break
-    if os.path.exists(CONFIG_FILE):
-        logging.info(
-            "Found Config File %s . NOTE: SETTINGS IN CONFIG FILE OVERRIDE COMMAND LINE OPTIONS",
```
Comment (darwayne) [inline panther_analysis_tool/main.py:1780]: same here, can we keep this but just say what it is actually doing instead?

Comment (dekatzenel) [reply panther_analysis_tool/main.py:1780]: Maybe check if any command line options are provided, if it's not too difficult, and only mention the override if so

File: example_panther_config.yml (hunk 1 above)
Comment (eliskaachee) [reply example_panther_config.yml:3]: Okay, how about something like "Note that options in this file will be overwritten by options passed on the command line"?

File: panther_analysis_tool/main.py (hunk 2 above)
Comment (eliskaachee) [reply panther_analysis_tool/main.py:1780]: That's a good idea, how about something like "Found Config File ____ NOTE: COMMAND LINE OPTIONS WILL OVERWRITE SETTINGS IN CONFIG FILE"

For checking if any command line options are provided, is that just checking "if args" from line 1758? Or do we just want to check for specific args?

File: example_panther_config.yml (hunk 1 above)
Comment (dekatzenel) [reply example_panther_config.yml:3]: Nit: s/overwritten/overridden but otherwise LGTM

File: panther_analysis_tool/main.py (hunk 2 above)
Comment (dekatzenel) [reply panther_analysis_tool/main.py:1780]: > how about something like "Found Config File ____ NOTE: COMMAND LINE OPTIONS WILL OVERWRITE SETTINGS IN CONFIG FILE"
same overwrite vs override switch but LGTM
> For checking if any command line options are provided, is that just checking "if args" from line 1758? Or do we just want to check for specific args?
Looks like args includes the subcommand which isn't in the config file, so you'd need to check if it has more args than that

**Review by dekatzenel — APPROVED** [review]

**Pull request merged**